integration/tests/api/docker_test.go

```diff
@@ -253,6 +253,9 @@ func TestDockerContainerNetworkStats(t *testing.T) {
 	containerId := fm.Docker().RunBusybox("watch", "-n1", "wget", "https://www.google.com/")
 	waitForContainer(containerId, fm)
 
+	// Add a little sleep to let the request complete a few times
```

Comment:
What request? `docker run`? `waitForContainer` is supposed to ensure that the container is running right?
Are you trying to give some time for cadvisor to gather metrics?